tests/functional/test_validate_config.py

```diff
@@ -14,3 +23,96 @@ def test_srpm_command_for_path(upstream_or_distgit_path, tmp_path):
         call_real_packit_and_return_exit_code(
             parameters=["--debug", "validate-config", str(upstream_or_distgit_path)],
         )
+
+
+@pytest.fixture
+def dummy_local_project(tmp_path):
+    dummy = MagicMock()
+    dummy.working_dir = str(tmp_path)
+    return dummy
```

Comment:
I think you've made it a bit too complicated for yourself..;) Maybe we should be a bit more clear in our [contribution guide](https://github.com/packit/packit/blob/main/CONTRIBUTING.md).

a) We use https://github.com/flexmock/flexmock as our mocking tool so it would be better to use that one so we have one approach in our code base.
b) Not sure if you know, but for CLI testing, the `click` docs might be of help: https://click.palletsprojects.com/en/stable/testing/